feature-group-definitions/flexbox-gap.yml

```
@@ -1,3 +1,4 @@
 name: Flexbox gap
+description: The `gap` CSS property in a flexbox layout specifies the size of the gap between flex items and between flex lines.
```

"flex lines" feels a bit obscure to me, what do you think of this?

```suggestion
description: The `gap` CSS property in a flexbox layout sets the size of the space between flex items on the main axis (columns) and cross axis (rows).
```

---

I agree it's obscure, but using "items" for cross axis is incorrect. Flex items are things that are placed along the main axis. And, when the container is set to wrap, the things that are placed along the cross axis are flex lines.
So it's a choice of either being more technically correct, or easier to understand.
That being said, I think it ends up being equivalent. Flex lines aren't something you can see in the DOM or style with CSS, so any gap between flex lines will end up being the exact same gap between flex items.

Maybe something simpler?

```suggestion
description: The `gap` CSS property in a flexbox layout sets the size of the space between items.
```

This way we don't mention the 2 dimensions, which might actually be better because over in `flexbox.yaml` we say that it's a "one-dimensional layout system".

---

Yes, this shorter version is much better!